refinery/core/views.py

```diff
@@ -1091,3 +1093,23 @@ def patch(self, request, uuid):
         return Response(
             serializer.errors, status=status.HTTP_400_BAD_REQUEST
         )
+
+
+@api_view(['GET'])
+@staff_member_required
+def site_statistics(request, **kwargs):
+    site_statistics_type = kwargs.get("type")
+    response = HttpResponse()
```

Comment:
Should this be `rest_framework.response.Response` if it's a DRF API view?

Reply:
No, it doesn't have to be. In this case, it's simpler without. To return csv data with a DRF `Response` we would actually have to add a [third party library](https://www.django-rest-framework.org/api-guide/renderers/#csv) to get & use a `CSVRenderer`.

Reply:
OK then does it need to be an API view?

Reply:
The decorator ended up being superfluous